test/jasmine/tests/hover_label_test.js

```diff
@@ -4197,20 +4215,21 @@ describe('hover distance', function() {
             'the range of search for points to hover to Infinity (part 2)', function() {
             var gd = document.getElementById('graph');
             var evt = { xpx: 450, ypx: 155 };
-            Plotly.relayout(gd, 'hoverdistance', -1);
-
-            Fx.hover('graph', evt, 'xy');
+            Plotly.relayout(gd, 'hoverdistance', -1)
+            .then(function() {
+                Fx.hover('graph', evt, 'xy');
 
-            var hoverTrace = gd._hoverdata[0];
+                var hoverTrace = gd._hoverdata[0];
 
-            expect(hoverTrace.curveNumber).toEqual(0);
-            expect(hoverTrace.pointNumber).toEqual(1);
-            expect(hoverTrace.x).toEqual(2);
-            expect(hoverTrace.y).toEqual(3);
+                expect(hoverTrace.curveNumber).toEqual(0);
+                expect(hoverTrace.pointNumber).toEqual(1);
+                expect(hoverTrace.x).toEqual(2);
+                expect(hoverTrace.y).toEqual(3);
 
-            assertHoverLabelContent({
-                nums: '(2, 3)',
-                name: 'trace 0'
+                assertHoverLabelContent({
+                    nums: '(2, 3)',
+                    name: 'trace 0'
+                });
```

Comment:
`done`/`failTest`

Reply:
Done in https://github.com/plotly/plotly.js/pull/5340/commits/1694bb407d4138591eda35d31f4b4e176af17282.